tests/ci/cdk/cdk/aws_lc_ec2_test_framework_ci_stack.py

```diff
@@ -97,18 +96,18 @@ def __init__(self,
         security_group = ec2.SecurityGroup(self, id="{}-ec2-sg".format(id),
                           allow_all_outbound=True,
                           vpc=vpc,
-                          security_group_name='macos_arm_ec2_sg')
+                          security_group_name='codebuild_ec2_sg')
```

Comment:
What's the difference between `macos_arm_ec2_sg` and `codebuild_ec2_sg`?

Reply:
No difference, it was part of making the ec2 logic more applicable to all instances. `codebuild_ec2_sg` is now used for the graviton3 instance as well, so the original name didn't make much sense anymore.